src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -47,16 +47,23 @@ class WebsiteSearchController
      */
     private $twig;
 
+    /**
+     * @var string[]
+     */
+    private $additionalIndexes;
+
     public function __construct(
         SearchManagerInterface $searchManager,
         RequestAnalyzerInterface $requestAnalyzer,
         ParameterResolverInterface $parameterResolver,
-        Environment $twig
+        Environment $twig,
+        array $additionalIndexes
```

Comment:
For BC this needs to be:

```suggestion
        array $additionalIndexes = []
```